server/middleware/auth_test.go

```diff
@@ -133,6 +136,22 @@ func testHTTPRequest(t *testing.T, authorizationHeader string, statusCode int, e
 
 type mockQueryClient struct{}
 
+func (c *mockQueryClient) GetOracleRegistration(uniqueID, oracleAddr string) (*oracletypes.OracleRegistration, error) {
+	return nil, nil
+}
+
+func (c *mockQueryClient) GetLightBlock(height int64) (*tmtypes.LightBlock, error) {
+	return nil, nil
+}
+
+func (c *mockQueryClient) GetCdc() *codec.ProtoCodec {
+	return nil
+}
+
+func (c *mockQueryClient) GetChainID() string {
+	return ""
+}
+
```

Comment:
For now, these methods return `nil` or "". Because these are not deeply related to `auth_test.go`.